pytorch_lightning/loggers/logger.py

```diff
@@ -346,6 +346,17 @@ def __iter__(self):
         # if DummyLogger is substituting a logger collection, pretend it is empty
         yield from ()
 
+    def __getattr__(self, name: str) -> Callable:
+        """Allows the DummyLogger to be called with arbitrary methods, to avoid AttributeErrors."""
+
+        def method(*args, **kwargs):
+            rank_zero_info(
+                f"The `DummyLogger` is being used and the method `{name}` was called on it but it does not exist."
+                " This is the case when `fast_dev_run` is enabled"
+            )
```

Comment:
What is the motivation behind emitting an info here? This would lead to a spam of messages for every logging step.
Also it is not only used for fast dev run. Can we drop it? A dummy method with return value None should suffice I think.

Reply:
I suggested this, because otherwise this could be silently being executed without users noticing

Reply:
I think we shouldn't do this. The whole point is that using the dummy logger silently makes all logging calls dumb and do nothing to simulate logging being turned off. If we need a message, it should be done through the trainer when fast dev run is active, IMO, we can say something along the lines of logging and checkpointing is suppressed for fast dev run.

Reply:
I rewrote it

Reply:
This looks great! Just the test needs to be adjusted